submissions/adekemiAwomolo/html-forms/contact-form.html

```diff
@@ -9,7 +9,15 @@
 	<!--header goes in here and navs-->
 	<header>
 		<nav>
+
```

Comment:
@kemysola Remove the extra empty lines here.

Reply:
@ToluAkin . I have corrected it ma, please kindly approve. Thank you ma.